tutorials/euclid_access/2_Euclid_intro_MER_catalog.md

```diff
@@ -134,29 +132,30 @@ df_s_irsa.head()
 - Plot the color-magnitude diagram
 
 ```{code-cell} ipython3
-mag_y_s_irsa=-2.5*np.log10(df_s_irsa["flux_y_templfit"]) + 23.9 # Y
-mag_h_s_irsa=-2.5*np.log10(df_s_irsa["flux_h_templfit"]) + 23.9 # H
+mag_y = -2.5 * np.log10(result_stars["flux_y_templfit"]) + 23.9 
```

Comment:
We could do a super simple `result_stars["flux_y_templfit"].to(u.ABmag)` here, but I run into a couple of bugs and issues while I was doing just that, so rescoped this PR to only address the pandas vs astropy usage.